quickwit/quickwit-actors/src/spawn_builder.rs

```diff
@@ -268,11 +269,24 @@ impl<A: Actor> ActorExecutionEnv<A> {
 
     async fn process_all_available_messages(&mut self) -> Result<(), ActorExitStatus> {
         self.yield_and_check_if_killed().await?;
-        let envelope = get_envelope(&mut self.inbox, &self.ctx).await;
```

Comment:
I ended up reworking my first fix because it added another pass in `process_one_message` for all the actors to yield, leading to some unit tests failing.